tests/functional/sync_sign_up.js

```diff
@@ -110,7 +110,12 @@ define([
         .switchToWindow('')
         .end()
 
-        .findByCssSelector('#fxa-confirm-header')
+        // give a small delay, the verification poll should not be occur,
+        // but if it is it will take a few seconds to complete. Add an
```

Comment:
nit `but if it is it will take a few seconds to complete` -> `but if it occurs then will take a few seconds to complete`